src/components/shared/NavBar.jsx

```diff
@@ -0,0 +1,64 @@
+import { useState } from 'react';
+import { Link } from 'react-router-dom';
+import Text from './typography/Text';
+import Navigation from '../../data/Navigation';
+import image from '../../assets/images/image.png';
+import Button from './Button';
+import Logo from './Logo';
+import Hamburger from './Hamburger';
+
+function NavBar() {
+	const { navItems, logo } = Navigation;
+	const [isNavOpen, setIsNavOpen] = useState(false);
+
+	const toggleNav = () => {
+		setIsNavOpen(!isNavOpen);
+	};
+
+	return (
+		<nav style={{ position: 'sticky', top: '0' }}>
+			<div
+				className='navbar flex justify-between items-center opacity-53 bg-hard-light bg-center bg-cover h-[73.8px] overflow-hidden pl-10 '
```

Comment:
![image](https://github.com/dscnitrourkela/project-udon/assets/100484401/9548d9af-da9e-4440-ac48-03435eb4840b)
Background is going to the back.